carbonmark-api/src/routes/listings/[id]/get.ts

```diff
@@ -0,0 +1,40 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from "fastify";
+import { gql_sdk } from "../../../utils/gqlSdk";
+import { getListingById } from "../../../utils/helpers/fetchMarketplaceListings";
+import { getTokenById } from "../../../utils/helpers/fetchPoolPricesAndStats";
+import { formatListing } from "../../../utils/marketplace.utils";
+import { Params, Querystring, schema } from "./get.schema";
+
+const handler = (fastify: FastifyInstance) =>
+  async function (
+    request: FastifyRequest<{
+      Params: Params;
+      Querystring: Querystring;
+    }>,
+    reply: FastifyReply
+  ) {
+    const { id } = request.params;
+    const sdk = gql_sdk(request.query.network);
+
+    const listing = (await getListingById(sdk, id)).listing;
```

Comment:
should 404 when no listing is found for given ID

Reply:
Done 👍